Week6/app/src/main/java/com/oliverperez/blockbusterchallenge/ui/login/LoginActivityPresenter.kt

```diff
@@ -0,0 +1,23 @@
+package com.oliverperez.blockbusterchallenge.ui.login
+
+import android.view.textclassifier.TextLinks
+import android.widget.TextView
+import com.oliverperez.blockbusterchallenge.model.LoginPrefs
+import kotlinx.android.synthetic.main.activity_login.*
+
+class LoginActivityPresenter(private val view: LoginActivity) {
```

Comment:
This is not how you should be doing MVP (if that's what you were trying to do) 
You always want to depend on abstractions(interface) not on concrete types (LoginActivity)